tools/generate-js.py

```diff
@@ -117,7 +117,7 @@ def generate_subtype(name: str):
 # We need types from bolt 4.
 csv_lines = []
 for boltnum in (4, 12):
-    with open('specs/bolt{}.csv'.format(boltnum), 'r') as f:
+    with open('../specs/bolt{}.csv'.format(boltnum), 'r') as f:
```

Comment:
Toplevel makefile (e.g. `make generated-bolt12.js`) assumes it's run in the toplevel directory, so this breaks?